session/bootstrap.go

```diff
@@ -477,11 +477,14 @@ const (
 	version66 = 66
 	// version67 restore all SQL bindings.
 	version67 = 67
-
-	// please make sure this is the largest version
-	currentBootstrapVersion = version67
 )
 
+// currentBootstrapVersion is modified to a function so we can hook its value for testing.
+// please make sure this is the largest version
+var currentBootstrapVersion = func() int64 {
```

Comment:
Why not define it as a non-const variable?

Reply:
Done